docs/configuration.rst

```diff
@@ -195,8 +195,18 @@ configuration file <https://docs.pycsw.org/en/latest/configuration.html>`_.
   directory of the GRASS GIS instalation, refered as `GISBASE
   <https://grass.osgeo.org/grass73/manuals/variables.html>`_
 
+[daemon]
+--------
+
+:pidfile:
```

Comment:
Not used?

Reply:
updated docs. skipped `pidfile` in current implementation.